code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -68,7 +111,16 @@
 	wrapped = null
 	//update_icon()
 
-/obj/item/weapon/gripper/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
+/obj/item/weapon/gripper/attack(mob/living/carbon/C as mob)
+	//Medical Grippers can shape people awake
+	if(istype(src,/obj/item/weapon/gripper/medical))
```

Comment:
First, you are using an istype in a proc that can be overriden.
Second, do you have any idea what removing the user argument will do?